rpc/openapi/openapi.yaml

```diff
@@ -1103,18 +1103,25 @@ paths:
     get:
       summary: Generates a data commitment for a range of blocks
       description: |
-        Generates a data commitment over an ordered list of blocks matching the query.
-
-        See /subscribe for the query syntax.
+        Generates a data commitment over an ordered list of blocks matching the provided range.
```

Comment:
[optional] would it help if this specified if the range is inclusive or exclusive of the first_block / last_block?

Reply:
like this? [641b8f4](https://github.com/celestiaorg/celestia-core/pull/981/commits/641b8f45dd5f2e49c0568932b88122aa7a56cc39)